src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingDiagnosticValidationPass.cs

```diff
@@ -82,11 +82,11 @@ private class LocationIgnoringDiagnosticComparer : IEqualityComparer<RazorDiagno
         {
             public static IEqualityComparer<RazorDiagnostic> Instance = new LocationIgnoringDiagnosticComparer();
 
-            public bool Equals(RazorDiagnostic x, RazorDiagnostic y)
-                => x is not null &&
-                    y is not null &&
-                    x.Severity.Equals(y.Severity) &&
-                    x.Id.Equals(y.Id);
+            public bool Equals(RazorDiagnostic? x, RazorDiagnostic? y)
+                => x is not null
+                && y is not null
+                && x.Severity.Equals(y.Severity)
+                && x.Id.Equals(y.Id);
```

Comment:
Personally I prefer the operators at the end of the line, and it seems you do too, except where it helps lining up the code.. but then the inconsistency is surprising, anthropologically. How do you feel about just ignoring whitespace rules slightly to line these things up, instead?

```suggestion
                => x is not null &&
                   y is not null &&
                   x.Severity.Equals(y.Severity) &&
                   x.Id.Equals(y.Id);
```

Reply:
> but then the inconsistency is surprising, anthropologically

I don't see it that way. The consistency I'm going for is lining up the start of the expressions. From that perspective, my style tends to be quite consistent. :smile:

Happy to make this change if you feel strongly about it.

Reply:
Upon further reflection, I went ahead updated all of the formatting to use your whitespace suggestion.